components/fireside/post/like/widget/widget.vue

```diff
@@ -1,21 +1,41 @@
 <template>
-	<span class="fireside-post-like-widget" v-app-auth-required>
-		<app-button
-			:icon="isCircle ? 'heart' : undefined"
-			:circle="isCircle"
-			:overlay="overlay"
+	<span class="fireside-post-like-widget">
+		<app-popper
+			trigger="manual"
+			:show="isShowingFollowPopover"
 			:block="block"
-			:primary="!!post.user_like || !isCircle"
-			:solid="!!post.user_like"
-			:badge="badge"
-			v-app-tooltip="tooltip"
-			@click="toggleLike"
+			@hide="isShowingFollowPopover = false"
+			@auto-hide="onFollowPopoverDismissed"
 		>
-			<span v-if="!isCircle">
-				<translate v-if="!post.user_like">Like This Post</translate>
-				<translate v-else>Liked</translate>
-			</span>
-		</app-button>
+			<app-button
+				:icon="isCircle ? 'heart' : undefined"
+				:circle="isCircle"
+				:overlay="overlay"
+				:block="block"
+				:primary="!!post.user_like || !isCircle"
+				:solid="!!post.user_like"
+				:badge="badge"
+				v-app-tooltip="tooltip"
+				v-app-auth-required
+				@click="toggleLike"
+			>
+				<span v-if="!isCircle">
+					<translate v-if="!post.user_like">Like This Post</translate>
+					<translate v-else>Liked</translate>
+				</span>
+			</app-button>
+
+			<div slot="popover" class="well fill-darkest">
+				<p class="small">
+					<translate>
+						Would you also like to follow this author?
```

Comment:
Let's just say "user" instead of "author" since that's what we use everywhere else